app/assets/stylesheets/report.scss

```diff
@@ -0,0 +1,34 @@
+.report {
+  background: #fff;
+  border-radius: 2px;
+  box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
+  display: block;
+  float: left;
+  margin: 1rem;
+  min-height: 300px;
+  position: relative;
+  transition: all 0.2s ease-in-out;
+  width: 100%;
+  
+  &:hover {
+    box-shadow: 0 10px 20px rgba(0,0,0,0.19), 0 6px 6px rgba(0,0,0,0.23);
```

Comment:
Color literals like `rgba(0, 0, 0, 0.19)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `rgba(0, 0, 0, 0.23)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Commas in function arguments should be followed by a single space